cooltools/expected.py

```diff
@@ -311,6 +382,37 @@ def cis_expected(
     Dataframe of diagonal statistics, indexed by region and diagonal number
 
     """
+    warnings.warn(
+        "`cooltools.expected.cis_expected()` is deprecated in 0.3.2, will be removed subsequently. "
+        "Use `cooltools.expected.diagsum()` and `cooltools.expected.diagsum_asymm()` instead.",
+        category=FutureWarning,
+        stacklevel=2,
+    )
+
+    def _bg2slice_frame(bg2, region1, region2):
```

Comment:
as we know with @nvictus  `cis_expected` doesn't do a good job at slicing pixels - at least it does it differently than `cooler.matrix().fetch()` - it's all because of `_bg2slice_frame` - which is why I made it internal function - and basically prepared the whole `cis_expected` for removal with the next release